tethysext/atcore/services/resource_spatial_manager.py

```diff
@@ -1,33 +1,42 @@
 """
 ********************************************************************************
-* Name: model_db_spatial_manager
-* Author: nswain
-* Created On: July 06, 2018
-* Updated on: December 19, 2018
-* Copyright: (c) Aquaveo 2018
+* Name: resource_spatial_manager
+* Author: nswain, msouffront & htran
+* Created On: December 15, 2020
+* Updated on: December 15, 2020
+* Copyright: (c) Aquaveo 2020
 ********************************************************************************
 """
 import os
 from jinja2 import Template
 
-from abc import abstractmethod
-
-from tethysext.atcore.services.exceptions import UnitsNotFound, UnknownUnits
 from tethysext.atcore.services.base_spatial_manager import BaseSpatialManager
-from tethysext.atcore.models.app_users import Resource
 
 
 class ResourceSpatialManager(BaseSpatialManager):
     """
-    Class for Spatial Managers using a postgres database.
+    A generic SpatialManger for SpatialResource.
     """
 
     SQL_PATH = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'resources', 'sql_templates')
     SLD_PATH = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'resources', 'sld_templates')
 
     # Vector Layer Types
     VL_EXTENT_VIEW = 'resource_extent_layer_view'
-    VL_EXTENT_TEMPLATE = 'resource_extent_layer_style'
+    VL_EXTENT_STYLE = 'resource_extent_layer_style'
+
+    def get_extent_layer_name(self, resource_id):
+        """
+        Get name of the extent layer for a given resource.
+
+        Args:
+            resource_id(str): id of the Resource.
+
+        Returns:
+            str: name of the extent layer.
+
+        """
+        return f'app_users_resources_{resource_id}'
```

Comment:
Please change 'app_users_resources_' to 'app_users_resources_extent_'. This will future proof the layer name if we end up having other layers from the app_users_resources table.